### ISSUE TITLE
Alternative Checkpoint Point selection missing from Hires options menu compared to Automatic 1111 

### ISSUE THREAD

**Issue by Mirabilis1729** [lllyasviel/stable-diffusion-webui-forge#1110]
Hi @lllyasviel

One issue I have noticed is the '**Hires fix: show hires checkpoint**'  which looks like this in A1111: 
 
<img width="919" alt="vivaldi_n9lGSgXQJt" src="https://github.com/user-attachments/assets/436eac19-91e5-4440-b91d-6f27ea5b1653">

Doesn't show up in Forge:  

<img width="909" alt="vivaldi_I3Z7tzyrPY" src="https://github.com/user-attachments/assets/3c323a28-0798-44b6-a5ac-c890cd9f17eb">

Even if you have it enabled in the uI settings.  

<img width="878" alt="vivaldi_hmhS66CdD8" src="https://github.com/user-attachments/assets/14e769cd-42d8-4fa7-99df-a5d5df3e32cf">

Might not seem like an obvious need, but I've found it quite useful to be able to switch over to a different model at times in order to introduces a more stylistic look to outputs at the upscale stage and it would be very cool to see in Forge if possible.  

![00392-RFKTRS-SP-V3-AA-V6-KUV-V1-05_,2656391738_ Euler_ 7_ 28_1080x1344](https://github.com/user-attachments/assets/26b51ea5-2243-4095-9049-38c6f4c3c788)
![00385-RFKTRS-SP-V3-AA-V6-KUV-V1-05_,3826165290_ Euler_ 7_ 28_1080x1344](https://github.com/user-attachments/assets/3307e8fa-cd6c-460d-ad37-0e87f0cc14c7)

_Originally posted by @Mirabilis1729 in https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/981#discussioncomment-10306474_

**Comment by andy8992** [comment]
FIgured... they'd have done something about this by now. Very annoying heh

**Comment by Mirabilis1729** [comment]
> FIgured... they'd have done something about this by now. Very annoying heh

Yeah it's not a feature I think a lot of people are aware of in A1111 as it's not on by default, but it does allow you to generate some genuinely unique outputs, as well as bridge model types, so you might start a generation in SD1.5 but then have the high res version output with SDXL or potentially FLUX or vise versa, esp given you can change the prompt with the second stage
